TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/README.md

```diff
@@ -61,58 +63,119 @@ Exactly one of the following blocks must be present:
 
 Having a `source_cluster` and `target_cluster` is required.
 
-#### Metrics Source
+### Metrics Source
 
 Currently, the two supported metrics source types are `prometheus` and `cloudwatch`.
 Exactly one of the following blocks must be present:
+
 - `prometheus`:
     - `endpoint`: required
 
 - `cloudwatch`: may be empty if region is not specified
     - `aws_region`:  optional. if not provided, the usual rules are followed for determining aws region. (`AWS_DEFAULT_REGION`, `~/.aws/config`, etc.)
 
+### Backfill
+
+Backfill can be performed via several mechansims. The primary two supported by the console library are Reindex From Snapshot (RFS) and
+OpenSearch Ingestion Pipeline (OSI).
```

Comment:
Do we want to provide any context on why we offer both or what the difference is?

Reply:
Yes, but I don't think this is the place to do it. By the time you're here, you're really deep in the weeds.